app/[lang]/GlobalRedux/store.js

```diff
@@ -1,33 +1,6 @@
 "use client";
 import { configureStore, createSlice } from "@reduxjs/toolkit";
 
-const useCartEffect = () => {
```

Comment:
Usuwam nieużywany kod z reduxa. Prawdopodobnie ten pierwszy useEffect wywalając się zaburzał pracę reduxa.

Po weryfikacji wychodzi na to, że stan jest dobrze trzymany i nie czyści się przy przejściu na koszyk. Nie widzieliśmy zmian w nim, bo mieliśmy w narzędziach do debuggowania ustawioną złą przestrzeń.
![image](https://github.com/dariuszstawik/smarteq/assets/43916042/8813d76f-ce90-4858-b503-2817fe527eed)